src/status_im/ui/screens/home/styles.cljs

```diff
@@ -185,13 +185,15 @@
    :margin-horizontal 32
    :color             colors/gray})
 
-(def action-button-container
+(defn action-button-container [home-width]
```

Comment:
could you elaborate on changes in this function? why do we need z-index and left?

Reply:
yes.. the + action-button was centered using `:align-items :center` of the parent which required all the children of the parent to have `:align-items :stretch` which seemed cumbersome and error prone(was puzzled at first when an element did not render without stretch), So, instead centered it using calculation.

I put a border around `connectivity-view-wrapper` for testing and previous method of centering would cause the whole content area to be briefly rendered squeezed(not stretched) when landing on home screen as well - which I thought was weird.

Reply:
critical part of how the new implementation works is by layering of sibling components using z-index and defying natural layering of bottom siblings being rendered on top of one above. So, in order for the action button to be above content, it required higher z-index.